pyamo/cli.py

```diff
@@ -236,7 +239,7 @@ def cmd_adminstatus(handler, amo, args):
 
     if args.file:
         print("Last chance to bail out before changes are made (Ctrl+C to quit, enter to continue)")
-        raw_input()
+        six.moves.input()
```

Comment:
You use six.moves.input() here, but input() in other places. Any specific reason?

Reply:
No specific reason, it would crash if control reached there.  I just forgot to change there. In fact the lib is pretty untested, I just ran it and fixed where I get exceptions, for now I haven't ran it further than the first `input` (I wonder if I really need an API key just to download the info about addons). Anyway, you can cherry-pick the first 2 commits, if they seem good to you.